include/crocoddyl/multibody/actuations/floating-base.hpp

```diff
@@ -50,11 +51,11 @@ class ActuationModelFloatingBaseTpl : public ActuationModelAbstractTpl<_Scalar>
     assert_pretty(data->dtau_dx == MatrixXs::Zero(state_->get_nv(), state_->get_ndx()), "dtau_dx has wrong value");
     assert_pretty(data->dtau_du == dtau_du_, "dtau_du has wrong value");
   };
+  boost::shared_ptr<StateMultibody> state = boost::dynamic_pointer_cast<StateMultibody> (state_);
```

Comment:
1) actuations/floating-base should not have been changed in this PR. Please revert these changes.

2) All your changes in this file look like they were introduced while debugging. This should have been cleaned before introducing the PR.

3) I'm surprised that the PR is even compiling with the changes that you put here!